onnxruntime/python/tools/quantization/calibrate.py

```diff
@@ -553,8 +579,16 @@ def compute_percentile(self):
             hist_edges = histogram[1]
             total = hist.sum()
             cdf = np.cumsum(hist/total)
-            idx = np.searchsorted(cdf, percentile/100)
-            thresholds_dict[tensor] = (float(hist_edges[idx]), float(hist_edges[idx]))
+            idx_right = np.searchsorted(cdf, percentile/100)
+            if self.symmetric:
+                thresholds_dict[tensor] = (-float(hist_edges[idx_right]), float(hist_edges[idx_right]))
+            else:
+                idx_left = np.searchsorted(cdf, (1.0 - percentile/100))
+                thresholds_dict[tensor] = (float(hist_edges[idx_left]), float(hist_edges[idx_right]))
```

Comment:
should it be left 1.0 - percentile/200 and right: percentile/200?

Reply:
Yes, fixed the code.